File: examples/ex3.c
#include <stdio.h>
#include <stdbool.h>
#include <string.h>

int MojaTablica[5 + 1];
int i;
int j;
int temp;


int main() {
    MojaTablica[1] = 42;
    MojaTablica[2] = 12;
    MojaTablica[3] = 89;
    MojaTablica[4] = 23;
    MojaTablica[5] = 11;
    for (i = 1; i <= 4; i++) {
        for (j = 1; j <= 5 - i; j++) {
            if (MojaTablica[j] > MojaTablica[j + 1]) {
                temp = MojaTablica[j];
                MojaTablica[j] = MojaTablica[j + 1];
                MojaTablica[j + 1] = temp;
            }
        }
    }
    printf("Posortowana tablica:\n");
    for (i = 1; i <= 5; i++) {
        printf("%d ", MojaTablica[i]);
    }
    return 0;
}

File: examples/ex3.pas
program SortowanieTablicy;

var
  MojaTablica : array[1..5] of Integer;
  i, j, temp  : Integer;

begin
  MojaTablica[1] := 42;
  MojaTablica[2] := 12;
  MojaTablica[3] := 89;
  MojaTablica[4] := 23;
  MojaTablica[5] := 11;

  for i := 1 to 4 do
  begin
    for j := 1 to 5 - i do
    begin
      if MojaTablica[j] > MojaTablica[j + 1] then
      begin
        temp := MojaTablica[j];
        MojaTablica[j] := MojaTablica[j + 1];
        MojaTablica[j + 1] := temp;
      end;
    end;
  end;
  writeln('Posortowana tablica:');

  for i := 1 to 5 do
    write(MojaTablica[i], ' ');
  
end.